hardhat.config.ts

```diff
@@ -51,15 +49,15 @@ const customAccounts = [
   `0xdf57089febbacf7ba0bc227dafbffa9fc08a93fdc68e1e42411a14efcf23656e`,
 ]
 
-let hardhatAccounts = []
-customAccounts.forEach(a => {
-  hardhatAccounts.push(
-    {
-      privateKey: a,
-      balance: "10000000000000000000000"
-    }
-  )
-});
+// let hardhatAccounts = []
```

Comment:
Feel free to just delete anything you think is not required.  It was all here when I took over the repo and I never fully understood exactly what it's for.  I guess we just need to ensure those predetermined accounts in the README are pre-funded.